### ISSUE TITLE
Fork geeknote?

### ISSUE THREAD

**Issue by SanketDG** [VitaliyRodnenko/geeknote#363]
geeknote is some awesome software and I am sad to see it being so inactive. Given the community around it is still active, would a fork be possible? A :+1: on this post would help. Interested people who want to take this forward can reply here.

We can then discuss how to make this happen seamlessly.

CC @VitaliyRodnenko

**Comment by sean-** [comment]
It'd be nice to know what the fate of this software is one way or another.  I'd personally be interested in seeing this rewritten in Go because `go get -u .../geeknotev2` would be all around epic.

**Comment by jeffkowalski** [comment]
I've been progressing [my fork](https://github.com/jeffkowalski/geeknote/network) fairly regularly.  It has substantial new features like resource uploading (i.e. attachments), deduplication, and a TravisCI framework for continuous testing.  Pull requests are certainly welcome.

**Comment by SanketDG** [comment]
@jeffkowalski Would you like to move your fork to a new organization? Creating a organization could help us organize better and also encourage new contributors.

Currently there are a lot of pending PRs on this repo that will need to be assessed.

We will also be needing the pypi rights but that depends on @VitaliyRodnenko

**Comment by jeffkowalski** [comment]
I think most, if not all of the PRs from Vitaliy's queue have been pulled into my fork.
Personally, I'm not interested in creating a new organization for the work, because that'd leave the other contributors behind, and would also disconnect the history, which is useful for debugging, as well as crediting the authors.  As well, it's already referenced from Arch Linux as [geeknote improved](https://aur.archlinux.org/packages/geeknote-improved-git/), so moving to a different repo would only create confusion.

**Comment by SilkAndSlug** [comment]
@jeffkowalski Apologies for the necro. I have a possible solution to the new repo/lost history problem: add the old repo as a remote, and fast-forward merge it into the new. I imagine Arch Linux could update their pointers.

**Comment by jeffkowalski** [comment]
I'm still struggling to understand the benefit of the proposal.
Is there something that we can't already accomplish in the current fork?
Why do we need a new organization to host a replica?
An example of something you're concretely trying to do today would be
helpful.

**Comment by SilkAndSlug** [comment]
@jeffkowalski : your fork has a bus-number of one, which is an issue in and of itself. If/when you disappear, someone else will(?) fork yours but... that's little use to those who are still tracking the old branch -- which was my problem, yesterday, tracking Vitaliy's branch. If you're happy with both of those points, rock on.

**Comment by jeffkowalski** [comment]
I'm sorry to not understand.  What do you mean by bus-number, and what's
the associated problem in tracking, exactly?  My branch is fully visible,
accepts PRs, has an issue list, etc.
What would be the difference if it were a fully separate repo?

**Comment by SilkAndSlug** [comment]
@jeffkowalski : 
**Bus Number** is the (smallest) number of people who'd need to be hit by a bus to kill the project. In the case of your fork, that's you: one.

**Comment by SilkAndSlug** [comment]
@jeffkowalski 

The 'problem in tracking' is that Git(Hub) doesn't have a mechanism to say "*this* fork has died; use *that* fork instead".

If Alice or Bob hit `git pull origin` they get the tip of Vitaliy's (dead?) fork, not yours -- nor whichever fork(s) replace yours in due course. Yes, they could go to GitHub, look at the Network Graph, choose a winner, and change their origin... but Joe SixPack couldn't. *His* copy of Geeknote doesn't get updates, and that's that.

Moving your fork to an [Organisation](https://github.com/blog/674-introducing-organizations) would allow you to hand the Repo over to your successor, without changing its URL. It'd also allow you to - if that's what you wanted - get help merging PRs, etc.

**Comment by SilkAndSlug** [comment]
@jeffkowalski 

By "if you're happy, rock on", I mean that I have no dog in this fight -- I'm just a user. My hands are full elsewhere.  😀

**Comment by SanketDG** [comment]
@SilkAndSlug I totally agree with the above points. I hate the Evernote browser app, it sucks. `geeknote` is an excellent alternative! But the fact that it's ignored makes me sad. There are so many awesome pulls, but none of that is merged in. @jeffkowalski's work is commendable, but moving the repo to an org, would greatly benefit the community.

I mean I bet there is one contributor, who is going like, "yeah this repo is inactive, I am just going to keep this awesome feature I implemented to myself." Nobody would care about pulling to a fork.

All I need is one more person who's with me on this because @SilkAndSlug said, I dont want to have an oragnization created by me with a bus number of one. I need this awesome community.

**Comment by GiacomoLaw** [comment]
I'd be happy to help. :smile:

**Comment by dotarmin** [comment]
What's the status of this project and this thread? Is anybody going for an "official" fork? 
It's really sad this project is not active any more because it's the best Evernote client/integration I've used.
